lib/block_view.go

```diff
@@ -3257,23 +3356,21 @@ func (bav *UtxoView) ValidateDiamondsAndGetNumDeSoNanos(
 	return desoToTransferNanos, netNewDiamonds, nil
 }
 
-func (bav *UtxoView) ConnectTransaction(txn *MsgDeSoTxn, txHash *BlockHash,
-	txnSizeBytes int64,
-	blockHeight uint32, verifySignatures bool, ignoreUtxos bool) (
-	_utxoOps []*UtxoOperation, _totalInput uint64, _totalOutput uint64,
-	_fees uint64, _err error) {
+func (bav *UtxoView) ConnectTransaction(
+	txn *MsgDeSoTxn, txHash *BlockHash, txnSizeBytes int64,
+	blockHeight uint32, blockTimestamp int64, verifySignatures bool,
```

Comment:
Always include the units of measurement in variable/parameter names. The ambiguity can lead to bugs down the line if someone misinterprets the units
```suggestion
	blockHeight uint32, blockTimestampNanoSecs int64, verifySignatures bool,
```